textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/utils/TestUtils.java

```diff
@@ -32,6 +38,40 @@ public static boolean contains(List<ITuple> sampleTuples, ITuple actualTuple, Li
         return contains;
     }
     
+    public static boolean containsAllResults(List<ITuple> expectedResults, List<ITuple> exactResults) {
+        if(expectedResults.size() != exactResults.size())
+        	return false;
+        if(!(expectedResults.containsAll(exactResults)) || !(exactResults.containsAll(expectedResults)))
+        	return false;
+        
+        return true;
+    }
+    
+    public static boolean checkResults(List<ITuple> results, String queryString, Analyzer queryAnalyzer, String searchField) throws ParseException {
+        
+    	
+    	boolean contains = false;
+        QueryParser parser = null;
+        parser = new QueryParser(searchField, queryAnalyzer);
+        Query query = parser.parse(queryString);
+        String searchString = query.toString(searchField);
+        System.out.println(query.toString(searchField));
+        String[] listOfQueryWords = searchString.split(" ");

```

Comment:
I saw the new containsAllResults() function done by @rajesh9625 . Still why don't we use the "queryAnalyzer" to tokenize the query string, instead of using  split(" ")?